docs/releases/ez_platform_v2.2.0.md

```diff
@@ -57,8 +57,7 @@ You can find the list of all bookmarks in *Browse content* section. There, you c
 
 ### Standard design
 
-eZ Platform now comes with two designs using the [design engine](../guide/design_engine.md): `standard` for content view and `admin` for the Back Office.
-See [default designs](../guide/design_engine.md#default-designs) for more information.
+eZ Platform now comes with two designs using the [design engine](../guide/content_rendering/design_engine/design_engine.md): `standard` for content view and `admin` for the Back Office.
```

Comment:
Somehow this change seems to not have been rendered in the build:
<img width="1190" alt="obraz" src="https://user-images.githubusercontent.com/64841871/121543000-dfe7ed80-ca08-11eb-82e0-b0c636c56caa.png">